network/manager.go

```diff
@@ -39,6 +39,7 @@ const (
 	ContainerIDLength    = 8
 	EndpointIfIndex      = 0 // Azure CNI supports only one interface
 	DefaultNetworkID     = "azure"
+	dummyGUID            = "12345678-1234-1234-1234-123456789012"
```

Comment:
can we add comments here why we need this dummyGUID?

Reply:
I'll add a comment.

I originally wanted to change UseHnsV2 to just return true if stateless is enabled. However, UseHnsV2 is called in both newNetwork and newEndpoint. In newNetwork, we have access to the network manager. In newEndpoint, we do not. Thus, I cannot add UseHnsV2 to network manager.

Two options remain:
1) Add dummy GUID during deletion. We want to always trigger UseHnsV2 to true in stateless deletion. UseHnsV2 is true when we have a guid in netns and the api is supported. So, we pass in a dummy guid to satisfy the first condition, and the second condition is always true in stateless.

2) Add netns to stateless. When we convert the saved cns state we can repopulate the netns in the epInfo field. However, I do not see us using the netns value anywhere (only whether or not it is a valid guid-- the actual guid value doesn't seem to matter).

Reply:
thanks Alexander, as discussed with TM here, let's add a TODO here to fix this in next PR